adapters/balance.js

```diff
@@ -38,6 +38,14 @@ module.exports = function (mu, transports) {
 
 
 
+  function setMu (muInstance) {
+    transports.forEach(function (transport) {
+      transport.setMu(muInstance)
+    })
+  }
```

Comment:
why? Just saying, outbound should be something that calls `mu`.

Anyway, we can always change this later on.